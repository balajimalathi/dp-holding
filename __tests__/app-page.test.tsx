// @vitest-environment jsdom
import { describe, it, expect } from 'vitest'
import { render, screen } from '@testing-library/react'
import '@testing-library/jest-dom/vitest'
import Home from '@/app/(app)/page'
// import Home from '../app/(app)/page'

describe('Home Page', () => {
  it('renders without crashing', () => {
    render(<Home />)
    expect(screen.getByText('Federal Demat Holding')).toBeInTheDocument()
  })

  it('matches snapshot', () => {
    const { asFragment } = render(<Home />)
    expect(asFragment()).toMatchSnapshot()
  })
})