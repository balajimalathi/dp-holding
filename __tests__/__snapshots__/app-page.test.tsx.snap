// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Home Page > matches snapshot 1`] = `
<DocumentFragment>
  <p>
    Hi
  </p>
</DocumentFragment>
`;
